Update receivers README (#553)
open-telemetry/opentelemetry-collector: receiver/README.md

with detailed sample configurations [here](https://github.com/open-telemetry/opentelemetry-collector/blob/master/receiver/opencensusreceiver/testdata/config.yaml).

### Communicating over TLS
This receiver supports communication using Transport Layer Security (TLS). TLS

> can be configured by specifying a `tls-crendentials` object in the receiver

Reviewer: ```suggestion
can be configured by specifying a `tls_crendentials` object in the receiver
```

Author: err also a type: credentials

examples.

### Communicating over TLS
This receiver supports communication using Transport Layer Security (TLS), but
only using the gRPC protocol. It can be configured by specifying a

> `tls-crendentials` object in the gRPC receiver configuration.

Reviewer 2: Should this be `tls_crendentials`?

## <a name="vmmetrics"></a>VM Metrics Receiver
**Only metrics are supported.**

Collects metrics from the host operating system. This is applicable when the
OpenTelemetry Collector is running as an agent.
```yaml
receivers:
  vmmetrics:

>     scrape_interval: 10

Reviewer: ```suggestion
    scrape_interval: 10s
```

    metric_prefix: "testmetric"
    mount_point: "/proc"

>     # process_mount_point: "/data/proc" # Should only be used for Daemon Set within a container.

Reviewer 3: > Should only be used for Daemon Set within a container.

This isn't very clear.

Author: Updated slightly -- not an expert on this, appears to be related to differences between deployment models.

```

## <a name="zipkin"></a>Zipkin Receiver
**Only traces are supported.**

This receiver receives spans from Zipkin (V1 and V2) HTTP uploads and

> translates them into the internal span types that are then sent to the

Reviewer: ```suggestion
translates them into the internal span types that are then pushed into trace pipelines according to the configuration.
```
